pkg/e2e/compose_build_test.go

```diff
@@ -66,6 +66,21 @@ func TestLocalComposeBuild(t *testing.T) {
 		res.Assert(t, icmd.Expected{Out: `"FOO": "BAR"`})
 	})
 
+	t.Run("build with multiple build-args ", func(t *testing.T) {
+		for i := 0; i < 5; i++ {
```

Comment:
I don’t think we need a loop here, it’s a deterministic bug.

Reply:
That was just a desperate attempt to reproduce it 🥲